scripts/gitOpsApi.js

```diff
@@ -0,0 +1,429 @@
+/*
+Copyright 2021 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/inclusive-design/data-update-github/main/LICENSE
+*/
+
+// API for Github operations.
+
+"use strict";
+
+module.exports = {
```

Comment:
The structure of this API is looking great, we should add some tests for the contents of this file